### PR TITLE
Add ready button

### LINKED ISSUE #26: Play song
Clients should be able to tell the server that they are ready, then allow the server to tell them to start

### PR DESCRIPTION
**Pull request by jonchan51**
Closes #26

### CODE REVIEW

File: src/components/DuetRoom.tsx [hunk 1]
```diff
@@ -116,9 +114,37 @@ const DuetRoom: React.FC<{ maybeRoomId: string | null; isCreate: boolean }> = ({
   const friendId = getFriendId(roomState, playerId);
   const partsSelection = getPartsSelection(roomState);
 
+  // if timeToStart is not 0,
+  //   hide readybutton, partselection, and parts of room header, show number
+  //   show number countdown
+  // if timeToStart is 0 and playing, show waterfall, music, etc.
+  // if timeToStart is 0 and not playing, show the current stuff
+  const middleBox = () => {
```
Comment (gary-lgy) [inline src/components/DuetRoom.tsx:122]: rip can we use `if` and return early instead of nested ?:

File: src/components/DuetRoom.tsx [hunk 2]
```diff
@@ -31,55 +33,36 @@ const useStyles = makeStyles(theme => ({
   },
   box: {
     flexGrow: 100,
+    position: 'relative',
```
Comment (gary-lgy) [inline src/components/DuetRoom.tsx:36]: Why do we need this?

File: src/components/DuetRoom.tsx [hunk 3]
```diff
@@ -31,55 +33,36 @@ const useStyles = makeStyles(theme => ({
...
   },
   header: {
     flexGrow: 0,
   },
   piano: {
     flexGrow: 0,
   },
+  readyButton: {
```
Comment (gary-lgy) [inline src/components/DuetRoom.tsx:44]: Should we remove the absolute spacing in the future? If so can we put a TODO here

**Review by gary-lgy — APPROVED** [review]

File: src/components/DuetRoom.tsx (hunk 2 above)
Comment (jonchan51) [reply src/components/DuetRoom.tsx:36]: It's for the readybutton to have it's absolute position wrt this element

**Pull request merged**